src/main/proto/ga4gh/read_service.proto

```diff
@@ -91,6 +103,52 @@ message GetReadGroupSetRequest {
   string read_group_set_id = 1;
 }
 
+// ******************  /readgroups  *********************/
+
+// This request maps to the body of `POST /readgroups/search` as JSON.
+
+
+message SearchReadGroupsRequest {
+  // The dataset to search.
```

Comment:
Great, I removed `read_group_set_id` from the request and made `dataset_id` required and then added a list of read_group_ids to the `ReadGroupSet` message.